.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           EMAIL_HOST_USER: xxx
           EMAIL_HOST_PASSWORD: xxx
           SCALINGO_APP: access4all
+          DATAGOUV_API_KEY: xxx
```

Comment:
Should we set this in the test needing it instead? It's a bit surprising an optional env var is required in CI…